### PR TITLE
Sprintr Release 2023.07.27

### PR DESCRIPTION
**Pull request by ghost**
Scheduled release Thursday 27 July 2023

### CODE REVIEW

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
 ## 2023
 
+### July 27, 2023
+
+#### Improvements
+
+* In Mendix 10, we have combined the strength of Mendix Studio and Studio Pro - they are not separate anymore, so we have removed the last links from the Developer Portal to Studio.
+* We have improved the information regarding the sunset of [Mendix Stories](/developerportal/project-management/stories/) (remember, it's October 1)
+* We are preparing the platform for the [big upgrade of the feedback management experience](https://www.mendix.com/blog/mendix-release-10-0-a-new-era/#app-insights). The new feedback management and mini surveys are a part of our brand new product suite called App Insights. The date for this upgrade of feedback management is set to August 12. If you are using the current feedback management tool, you also received more detailed information in an email sent on Monday, July 24 (subject line: Upgrade to new Feedback Management).
+
```
Comment (ghost) [inline content/en/docs/releasenotes/developer-portal/_index.md:28]: Please add the following line:
You asked, we listened: In [Epics](https://docs.mendix.com/developerportal/project-management/epics/), you can now see who created a story on the story details page.

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
 
 ## 2023
 
+### July 27, 2023
+
+#### Improvements
+
+* In Mendix 10, we have combined the strength of Mendix Studio and Studio Pro - they are not separate anymore, so we have removed the last links from the Developer Portal to Studio.
+* We have improved the information regarding the sunset of [Mendix Stories](/developerportal/project-management/stories/) (remember, it's October 1)
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/developer-portal/_index.md:25]: @loket4 
Hi Olaf - should we put 2024 here as well, so as not to scare the customers?

File: content/en/docs/releasenotes/developer-portal/_index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/releasenotes/developer-portal/_index.md:28]:
```suggestion
* You asked, we listened: In Epics, you can now see who created a story on the story details page.

```

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 3]
```diff
@@ -17,6 +17,14 @@ To see the current status of the Mendix Developer Portal and Control Center, see
...
+* We are preparing the platform for the [big upgrade of the feedback management experience](https://www.mendix.com/blog/mendix-release-10-0-a-new-era/#app-insights). The new feedback management and mini surveys are a part of our brand new product suite called App Insights. The date for this upgrade of feedback management is set to August 12. If you are using the current feedback management tool, you also received more detailed information in an email sent on Monday, July 24 (subject line: Upgrade to new Feedback Management).
```
Comment (MarkvanMents) [inline content/en/docs/releasenotes/developer-portal/_index.md]: @loket4 
Lakshya just told me August 13th.
```suggestion
* We are preparing the platform for the [big upgrade of the feedback management experience](https://www.mendix.com/blog/mendix-release-10-0-a-new-era/#app-insights). The new feedback management and mini surveys are a part of our brand new product suite called App Insights. The date for this upgrade of feedback management is set to August 13. If you are using the current feedback management tool, you will have received more detailed information in an email sent on Monday, July 24 (subject line: Upgrade to new Feedback Management).
```

**Review by MarkvanMents — COMMENTED** [review]
Hi Olaf (@loket4)
I've asked a couple of questions/made a couple of suggested changes.
Please can you let me know if they are good and tag me here, or in the #docs channel when the release is successful and I can publish.
Thanks.

File: content/en/docs/releasenotes/developer-portal/_index.md (hunk 2 above)
Comment (ghost) [reply content/en/docs/releasenotes/developer-portal/_index.md:25]: The functionality will be removed on 1 October 2023. The data will be available to customers until 1 October 2024. In thi case I would mention 1 October 2023.

Comment (MarkvanMents) [reply content/en/docs/releasenotes/developer-portal/_index.md:25]: OK, let's add 2023 to this note just to be absolutely clear!
```suggestion
* We have improved the information regarding the sunset of [Mendix Stories](/developerportal/project-management/stories/) (remember, it's October 1, 2023)
```

**Review by MarkvanMents — APPROVED** [review]
This can be published with or without my last proposed change.

**Comment by MarkvanMents** [comment]
Release confirmed over Slack.

**Pull request merged**